src/markov/markov_approx.jl

```diff
@@ -130,3 +130,50 @@ function _rouwenhorst(p::Real, q::Real, m::Real, Δ::Real, n::Integer)
         return linspace(m-Δ, m+Δ, n), θN
     end
 end
+
+
+"""
+Accepts the simulation of a discrete state Markov chain and estimates
+the transition probabilities via maximum likelihood.
```

Comment:
I've added a description of the likelihood -- It was a bit of a pain to do with unicode, but I think I did about as good as unicode will let me. (Julia docs don't support mathjax/latex, right?)

Thanks for links John. I've added those to documentation to point people in right direction if they want more info.